### PR TITLE
updated Admin URI Page

### PR DESCRIPTION
**Pull request by camhashemi**
added URI information, Aggregated Content information

### CODE REVIEW

**Comment by camhashemi** [comment]
@leogrim @stkem

File: kifi-backend/modules/shoebox/conf/shoeboxService.routes [hunk 1]
```diff
@@ -569,6 +569,8 @@ GET     /admin/pageInfos            @com.keepit.controllers.admin.AdminPageInfoC
 
 GET     /admin/scraped/:uriId       @com.keepit.controllers.admin.ScraperAdminController.getScraped(uriId: Id[NormalizedURI])
 GET     /admin/uri/:uriId           @com.keepit.controllers.admin.UrlController.getURIInfo(uriId: Id[NormalizedURI])
+#GET     /admin/uri/article/        @com.keepit.controllers.admin.UrlController.getArticle(articleKey: ArticleKey[_])
```
Comment (stkem) [inline kifi-backend/modules/shoebox/conf/shoeboxService.routes]: Intentionally left commented out?

Comment (camhashemi) [inline kifi-backend/modules/shoebox/conf/shoeboxService.routes]: Yes, it depends on what Leo just merged. Updated pull request on the way.

File: kifi-backend/modules/shoebox/marketing [hunk 2]
```diff
@@ -1 +1 @@
-Subproject commit f4df3a7fb3f6cee02d73f19ca48fcba969ce05c0
+Subproject commit 3a5e5c28af64e6adc0626eeffbdc4bc411a58050
```
Comment (ahsu1230) [inline kifi-backend/modules/shoebox/marketing]: watch out for this marketing submodule change unless you're actually working on the marketing site

Comment (camhashemi) [inline kifi-backend/modules/shoebox/marketing]: I'm definitely not. I'll make sure this isn't merged.

**Comment by camhashemi** [comment]
@leogrim should be good to go

**Pull request merged**